recipe/meta.yaml

```diff
@@ -28,7 +26,7 @@ build:
   # - nompi: osx issue introduced in 8.1, fixed in future 8.2
   # - openmpi: runtime failures during test suite
   skip: true  # [not linux]
-  string: {{ build_string }}
+  string: py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ mpi }}_{{ build }} 
```

Comment:
Is python required for `cp2k`? If it is then `python` should be a dependency on the `host` level. Alternatively, in case it is not required, then we should not use `py{{ CONDA_PY }}_*` in our build string and mark the build as `noarch: generic`.  https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-generic

Reply:
that's a good point - I believe it can be marked as `noarch`

https://github.com/cp2k/cp2k/blob/master/INSTALL.md#2b-python-required-build-system